.dvc/config

```diff
@@ -0,0 +1,8 @@
+[cache]
+    dir = /home/scil/.scilpy/dvc-cache/
```

Comment:
absolut path ?

Reply:
I will move the whole cache field inside of io/dvc, create it dynamically in python. The thing is this file doesn't expand environment variables, so I cannot assign $HOME and make the behavior similar to before.